Create loc-only-mt-engine-updates.md

JasonCard opened the pull request:
This INCLUDES file will remain empty for English repo, but used in CONTRIBUTING.md and contents will be added in loc repos to provide MT engine update for international contributors.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

JasonCard committed:
Create loc-only-mt-engine-updates.md

This INCLUDES file will remain empty for English repo, but contents will be added in loc repos to provide MT engine update for international contributors.

JasonCard committed:
Update CONTRIBUTING.md

This INCLUDES file will remain empty for English repo, but used in CONTRIBUTING.md and contents will be added in loc repos to provide MT engine update for international contributors.

JasonCard commented:
@tdykstra Thank you for the first approval. I've added the reference to the INCLUDES file in CONTRIBUTING.md. I also found that "Markdownlint failed due to blank lines in INCLUDES file, but can we have an exception for this PR?

tdykstra commented:
The last time I tried to ignore a linter warning it showed up in every subsequent PR.  One thing I've seen done to disable a rule for a specific file is to include a comment:
`<!-- markdownlint-disable MD012 -->`
But that may be specific to docs.ms.com articles, and the comment would show up in CONTRIBUTING.md.
You could try just putting a period in the line and that would be a minimal nuisance for CONTRIBUTING.md until/unless you put real content in it.

JasonCard committed:
Update loc-only-mt-engine-updates.md

Since INCLUDES cannot be empty, can this content be added for International content?

JasonCard commented:
@tdykstra Thank you for the suggestion. Since the INCLUDES file cannot be empty, I've added the real content we want to add for our international contributors. Would it be acceptable to add this content specific for international contributors? Please help to add any reviewers as appropriate.

mairaw commented on CONTRIBUTING.md:
```diff
@@ -21,6 +21,8 @@ This repository contains the conceptual documentation for .NET. The .NET documen
 - [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
     Issues and tasks for this repo are tracked in [dotnet/docs/issues](https://github.com/dotnet/docs/issues).
 
+[!INCLUDE [](includes/loc-only-mt-engine-updates.md)]
```
The contributing file is a repo only file that doesn't accept Markdown extensions. So this would show up for customers reading this.

JasonCard replied:
Do you mean that INCLUDES reference won't work for CONTRIBUTING.md, so I should directly add the content?

mairaw replied:
Yep. This what customers will see right now: https://github.com/dotnet/docs/blob/3b24abde90b95f531799197b392667706b0aa7c3/CONTRIBUTING.md

JasonCard replied:
I see, thank you. I've replaced the INCLUDES reference with proposed content and deleted the INCLUDES file from this PR.

JasonCard committed:
Update CONTRIBUTING.md

Changing INCLUDES reference to content

JasonCard committed:
Delete loc-only-mt-engine-updates.md

Removing INCLUDES file

tdykstra commented on CONTRIBUTING.md:
```diff
@@ -21,6 +21,10 @@ This repository contains the conceptual documentation for .NET. The .NET documen
 
+### Contributing to International content
+
+Contribution for Machine Translated (MT) content are currently not accepted and deactivated for now. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. ...
```
```suggestion
Contributions for Machine Translated (MT) content are currently not accepted. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. MT topics will have a disclaimer stating that part of the topic may be machine translated, and the **Edit** button won't be displayed because editing is disabled. 
```

tdykstra reviewed (commented):
I made a suggestion for a wording change, but one thing I wasn't sure how to clarify: initially we say "contributions are not accepted," then we say "over time the contributions will improve the quality..." implying that some contributions are being accepted.

JasonCard committed:
Update CONTRIBUTING.md

Clarifying based on review and suggestion.
